challenges/systems/challenge-2.md

```diff
@@ -0,0 +1,283 @@
+# Summary
+
+Implement a prototype CLI tool that can perform OIDC authentication.
```

Comment:
```suggestion
Implement a prototype CLI tool that can perform OIDC Authorization Code Flow with PKCE.
```

Reply:
Reworded, can you review and tell me if you think the new wording makes more sense?